option optimize_for = LITE_RUNTIME;

message CsRiverLanternNewFloatNotify
{
 repeated uint32 id = 1;
 repeated uint32 ts = 2;
 repeated uint32 num = 3;
}
